infrastructure/pcluster-manager.yaml

```diff
@@ -151,9 +158,9 @@ Resources:
           SITE_URL: !Sub
            - https://${Api}.execute-api.${AWS::Region}.${AWS::URLSuffix}
            - Api: !Ref ApiGateway
-          AUTH_PATH: !GetAtt [ PclusterManagerCognito, Outputs.UserPoolAuthDomain ]
-          SECRET_ID: !GetAtt [ PclusterManagerCognito, Outputs.UserPoolClientSecretName ]
-          AUDIENCE: !GetAtt [ PclusterManagerCognito, Outputs.AppClientId ]
+          AUTH_PATH: !If [ UsesCognito, !GetAtt [ PclusterManagerCognito, Outputs.UserPoolAuthDomain ], !Ref AWS::NoValue ]
+          SECRET_ID: !If [ UsesCognito, !GetAtt [ PclusterManagerCognito, Outputs.UserPoolClientSecretName ], !Ref AWS::NoValue ]
+          AUDIENCE: !If [ UsesCognito, !GetAtt [ PclusterManagerCognito, Outputs.AppClientId ], !Ref AWS::NoValue ]
```

Comment:
question: how do customers specify values for these ones when they select a different OIDC provider?

Reply:
We discussed it with @BarcoMasile and these values wouldn't be needed in the case of Okta (afaik the only one we would need is `AUDIENCE`, which in the case of Okta would be populated in the `PclusterApiHandle.py` file with the value taken from `CLIENT_ID`).

Reply:
great, thank you